### PR TITLE
[HO][FILE-489] re-arranged endpoints in order to improve the flow of …

### PR DESCRIPTION
**Pull request by habeeb01**
…the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,74 +337,6 @@ Response (in Body):
 }
 ```
 
-
-#### Download Zip
```
Comment (kastoestoramadus) [inline README.md]: why is it deleted?

File: README.md [hunk 2]
```diff
@@ -114,6 +115,24 @@ The following are the possible Envelope Statuses that occur for an Envelope.
 | CLOSED | A routing request has been made and files are "AVAILABLE". (The status "SEALED" will automatically changed to "CLOSED" once the file/s has become "AVAILABLE".) |
 | DELETED | Envelope has been deleted. |
 
+#### Soft Delete an Envelope (RECOMMENDED)
```
Comment (kastoestoramadus) [inline README.md]: soft deletion belongs unfortunately to transfer enpoints

File: README.md (hunk 1 above)
Comment (habeeb01) [reply README.md]: because it was duplicated

**Comment by kastoestoramadus** [comment]
:-1: soft delete belongs to routing

**Pull request closed without merging**